src/Platform.Handlers/src/Xamarin.Platform.Handlers/Platform/Android/SwitchExtensions.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Android.Graphics.Drawables;
+using Xamarin.Forms;
+using ASwitch = AndroidX.AppCompat.Widget.SwitchCompat;
+
+namespace Xamarin.Platform
+{
+	public static class SwitchExtensions
+	{
+		static bool ChangedThumbColor;
```

Comment:
I don't think this can be here. On the renderer this was a per instance value so this can't be a static shared across all the handlers

I wonder for the ThumbDrawable if we can use `SetTintList` on the drawable and define the different colors for those states

For API 23+ I think we can use these APIs

https://developer.android.com/reference/android/widget/Switch#setThumbTintList(android.content.res.ColorStateList)

https://developer.android.com/reference/android/widget/Switch#setTrackTintList(android.content.res.ColorStateList)